src/views/HomePageView/ExclusiveActivity/ExclusiveActivity.jsx

```diff
@@ -0,0 +1,70 @@
+// vendors
+import React from 'react';
+import GatsbyImage from 'gatsby-image';
+
+// utils
+import { graphql, useStaticQuery } from 'gatsby';
+
+// styles
+import {
+  ActivityWrapper,
+  activityImage,
+  ActivityIntro,
+  CTAWrapper,
+} from './ExclusiveActivity.styles';
+
+// components
+import Center from '../../../components/LayoutSections/Center';
+import Button from '../../../components/Button';
+
+const ExclusiveActivity = () => {
+  const data = useStaticQuery(
+    graphql`
+      query {
+        activityPicture: file(
+          relativePath: { eq: "img-exclusive-activity.jpg" }
+        ) {
+          childImageSharp {
+            fluid(
+              quality: 100
+              maxWidth: 1000
+              duotone: { highlight: "#EBEBEB", shadow: "#262626" }
+            ) {
+              ...GatsbyImageSharpFluid_withWebp_noBase64
+            }
+          }
+        }
+      }
+    `
+  );
+
+  const {
+    activityPicture: {
+      childImageSharp: { fluid: activityPicture },
+    },
+  } = data;
+
+  return (
+    <ActivityWrapper id='activite'>
```

Comment:
Tu peux ajouter un `h2` visually hidden ici pour respecter la sémantique dans la page et qui va servir pour l'a11y avec les lecteurs d'écran
```suggestion
    <ActivityWrapper id='activite'>
         <h2 css={hideVisually}>Activité exclusive</h2>
```